### DISCUSSION TITLE
Missing Records in Google Ads Source - ad_group vs. ad_group_criterion

### DISCUSSION THREAD

**Discussion by karolur** [airbytehq/airbyte#38091] [Connector Questions]
Hello,

I'm experiencing an issue with the Google Ads source connector where there appears to be a discrepancy in the number of records between the ad_group and ad_group_criterion streams.

Stream: ad_group
Expected Behavior: All ad groups from the Google Ads source should be included in this stream.
Actual Behavior: The number of unique ad_group_id values in the ad_group stream is approximately 70% less than the number of unique ad_group_id values in the ad_group_criterion stream.

This suggests that a significant portion of ad groups are missing from the ad_group stream. 

Additional Information:

    Helm Chart Version: 0.49.7
    Deployment: OSS GKE
    Source Connector: Google Ads (version 3.4.1)
    Destination Connector: BigQuery (version 2.3.7)
    
Any help would be appreciated or perhaps I should open an issue with this?

Thank you in advance.

**Comment by marcosmarxm** [comment]
duplicate of https://github.com/airbytehq/airbyte/issues/38092
